common/scripts/setup_env.sh

```diff
@@ -78,6 +78,8 @@ export TARGET_OUT_LINUX="${TARGET_OUT_LINUX:-$(pwd)/out/linux_amd64}"
 export CONTAINER_TARGET_OUT="${CONTAINER_TARGET_OUT:-/work/out/${TARGET_OS}_${TARGET_ARCH}}"
 export CONTAINER_TARGET_OUT_LINUX="${CONTAINER_TARGET_OUT_LINUX:-/work/out/linux_amd64}"
 
+export KUBECONFIG="/home/.kube/config"
```

Comment:
Should this be $HOME?

Reply:
The flow of .kube/config file into the build container is as follows

1. run make test
1. make, as part of its execution creates and runs a build container
1. the build container, as part of its execution, runs this script
1. the build container then executes the job

Inside the build container, KUBECONFIG is always `/home/.kube/config`

Atleast with, https://github.com/istio/tools/pull/899

Reply:
job^target